dpe/Cargo.toml

```diff
@@ -12,6 +12,8 @@ dpe_profile_p384_sha384 = []
 
 [dev-dependencies]
 asn1 = "0.13.0"
+bytemuck = {version = "1.13", features = ["derive"]}
```

Comment:
FYI: we're using zerocopy in the boot ROM; might want to standardize on that if you don't have a preference.

Reply:
Good to know, I switched it to use `zerocopy`. I think it would be best to be consistent.